Apps/W1/ServiceDeclaration/app/src/ServiceDocuments/ServDeclServLine.TableExt.al

```diff
@@ -34,11 +33,11 @@ tableextension 5034 "Serv. Decl. Serv. Line" extends "Service Line"
 
             trigger OnValidate()
             var
-                SalesHeader: Record "Sales Header";
+                ServiceHeader: Record "Service Header";
             begin
-                SalesHeader.Get("Document Type", "Document No.");
+                ServiceHeader := Rec.GetServHeader();
```

Comment:
It is better to check first whether `Applicable For Serv. Decl.` is `true` before requesting the service header? Like this:
```al
begin
    if "Applicable For Serv. Decl." then begin
        ServiceHeader := Rec.GetServHeader();
        ServiceHeader.TestField("Applicable For Serv. Decl.");
    end:
end;
```

Reply:
That is true. I've processed the PR "as is", but will make that change in a follow-up PR.